app/models/request.rb

```diff
@@ -30,13 +30,13 @@ class Request < ApplicationRecord
   after_initialize :set_defaults
 
   def invalidate_number_of_children
-    update(:number_of_children => children.size)
+    update(:number_of_children => requests.size)
```

Comment:
why not create an alias method `children`?

Reply:
@bzwei Because it doesn't reflect reality
The URL reads /requests/{id}/requests
https://github.com/RedHatInsights/approval-api/blob/bf3bd885938f791012835d660e166e5057e96119/public/doc/openapi-3-v1.2.json#L331
But internally you use children, when the graphql parser generates the schema from the openapi.json it doesn't know what children is. So this is needed for consistency

Reply:
that's exactly what I meant. We can use children as an alias method.

Reply:
@bzwei Remove extra to keep codes simple